### PR TITLE
Added `kustomize` target as dependency for `test-e2e` target

### PR DESCRIPTION
**Pull request by Bslabe123**
#### What type of PR is this?
/kind bug

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -121,7 +121,7 @@ test-integration: manifests fmt vet envtest ginkgo ## Run integration tests.
 	$(GINKGO) --junit-report=junit.xml --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
 
 .PHONY: test-e2e
-test-e2e: manifests generate fmt vet envtest ginkgo kind-image-build
+test-e2e: manifests generate fmt vet envtest ginkgo kustomize kind-image-build
```
Comment (liurupeng) [inline Makefile:124]: remove "generate"

Comment (liurupeng) [inline Makefile:124]: in line 225, please install LeaderWorkerSet manifests in "Kustomize"

Comment (liurupeng) [inline Makefile:124]: 1. remove "generate", it is not needed
2. In e2e-test.sh, line 22, use "kubectl" instead of "$KUBECTL"

**Comment by kerthcet** [comment]
Seems fixed by https://github.com/kubernetes-sigs/lws/pull/79, we can close this now.
/close

**Pull request closed without merging**